rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -164,13 +219,15 @@ def mock_requests_get(url: str, timeout: int):  # pylint: disable=unused-argumen
     assert 'shutter' not in data[TEST_ADDR2]
 
     # Test cache files are created
-    for protocol_name in AIRDROPS:
+    for protocol_name in ('uniswap', '1inch', 'grain', 'shapeshift', 'cow_gnosis', 'diva', 'shutter'):  # noqa: E501
```

Comment:
instead of listing them can't you maybe move the mock response to a variable and then take them from the variable here?